l10n_it_account_stamp/views/invoice_view.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <!-- Form: Invoice Customer -->
+        <record id="invoice_form_stamp" model="ir.ui.view">
+            <field name="name">account.invoice.form.stamp</field>
+            <field name="model">account.invoice</field>
+            <field name="inherit_id" ref="account.invoice_form"/>
+            <field name="arch" type="xml">
+
+                <xpath expr="//field[@name='invoice_line']/tree/field[@name='sequence']" position="after">
+                    <field name="is_stamp_line" invisible="1"/>
+                </xpath>
+
+                <xpath expr="//field[@name='invoice_line']/tree" position="attributes">
+                    <attribute name="colors">CadetBlue:is_stamp_line==True;</attribute>
+                </xpath>
+
+            </field>
+        </record>
+
+        <!-- Form: Invoice Supplier -->
```

Comment:
Questo modulo non serve solo per il bollo nelle fatture attive? Qual è il caso d'uso delle fatture passive?

Reply:
se un fornitore mi manda una fattura per cui va versato il bollo, ma lui erroneamente non lo fa, va fatto, per cui è utile che il sistema mi proponga in automatico il bollo, poi lo dovrò versare e farlo pagare al fornitore

Reply:
Una cosa importante da ricordare è, se serve, distinguere bill e invoice nelle stringhe sorgente.
Se la stringa è unica e va bene per entrambi i casi basta mettere "bill/invoice".

Reply:
> ma lui erroneamente non lo fa

@sergiocorato cosa non fa? Versarlo? Come faccio a saperlo?

Reply:
@sergiocorato quindi il caso è: il fornitore mi manda una fattura con un'imposta per cui dovrebbe versare il bollo, ma non valorizza DatiBollo dell'XML
?

In questo caso quindi io che ricevo la fattura devo versare il bollo? Non devo al massimo dire al fornitore di fare la fattura giusta?

Reply:
@elbati il bollo è a carico di chi riceve, per cui non so se il fornitore avendo dimenticato di mettere il bollo poi rifaccia la fattura, comunque è previsto nel caso delle fatture mediche, vedi Risoluzione n. 444 del 18/11/08:
`In particolare, ai sensi dell'articolo 22 del DPR n. 642 del 1972, la parte
cui viene consegnato un atto non in regola con le disposizioni sul bollo, alla
formazione del quale non abbia partecipato, entro quindici giorni dalla data del
ricevimento, deve presentare l'atto all'ufficio locale delle entrate e provvedere
alla sua regolarizzazione col pagamento della sola imposta.
In tale caso, la parte che ha provveduto alla regolarizzazione è esente da qualsiasi
responsabilità (sia ai fini del tributo che ai fini sanzionatori), mentre la sanzione è3
irrogata nei confronti del solo soggetto che ha formato l'atto senza assolvere
all'obbligo di pagare il tributo.`